src/components/ModeMenu.test.js

```diff
@@ -0,0 +1,16 @@
+import ModeMenu from './ModeMenu';
+import { renderComponent } from '../testsUtilities/renderComponent';
+describe('ModeMenu', () => {
+  it('Properly shows rendered component', () => {
+    renderComponent(ModeMenu());
+    const testedDiv = document.querySelector('.modeMenu');
+    const buttonList = [...document.querySelectorAll('.modeMenu__button')];
+    expect(testedDiv).toBeInTheDocument();
+    expect(buttonList.length === 3).toBeTruthy();
+    expect(
+      buttonList[0].textContent === 'Characters' &&
+        buttonList[1].textContent === 'Quotes' &&
+        buttonList[2].textContent === 'Deaths',
+    ).toBeTruthy();
```

Comment:
tutaj każdą linijkę proponowałbym zamienić na mniej więcej to:
`expect(screen.getByText('test button changed')).toBeTruthy();`
taką funkcję znajdziesz w pliku 'TestButton.test.js' i możesz sobie zerknąć jak to tam działa :)

Reply:
Jasne, poprawię.

Reply:
w TestButton.test.js jest wykorzystana:    
 _expect(testedComponent.innerText).toBe('test button clicked');_

screen.getByText rzuca mi błędem, że to nie jest funkcja. Wykorzystywałeś to gdzieś?